tests/unittests/text_prediction/test_metrics.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from autogluon.task.text_prediction.metrics import calculate_metric_by_expr
+
+
+def test_calculate_metric_by_expr():
```

Comment:
Does this support more complex operations? If so I'd recommend adding tests for them.